src/libraries/System.Text.RegularExpressions/src/System/Text/RegularExpressions/RegexParseError.cs

```diff
@@ -3,38 +3,39 @@
 
 namespace System.Text.RegularExpressions
 {
-    internal enum RegexParseError
+    public enum RegexParseError
```

Comment:
Another `///` documentation here would be great!

Reply:
The enum now has doc comments on every field and on the type.